### DISCUSSION TITLE
Feature Request: FAB - Floating Action Buttons & Speed Dial

### DISCUSSION THREAD

**Discussion by modbender** [saadeghi/daisyui#1677] [Ideas / feature requests]
## FAB Description:
A Floating Action Button (FAB) performs the primary, or most common, action on a screen.

A floating action button appears in front of all screen content, typically as a circular shape with an icon in its center. FABs come in two types: regular, and extended.

Only use a FAB if it is the most suitable way to present a screen's primary action. Only one component is recommended per screen to represent the most common action.

(Source: MUI - https://mui.com/material-ui/react-floating-action-button/)

## Speed Dial Description:
When pressed, a floating action button can display three to six related actions in the form of a Speed Dial.

(Source: MUI - https://mui.com/material-ui/react-speed-dial/)

**Comment by saadeghi** [comment]
It is currently possible by mixing other components:
- [`toast`](https://daisyui.com/components/toast/) puts an element on the corner of screen
- [`btn btn-circle`](https://daisyui.com/components/button/) for the circle button
- [`dropdown`](https://daisyui.com/components/dropdown/) for showing more items on click or hover

Here are examples: 
- https://play.tailwindcss.com/n6UYoWq4y5
- https://play.tailwindcss.com/RymHm7SohZ

**Reply by modbender** [reply]
Yup definitely possible, thank you. Though if you have spare time and if it is possible, do consider adding it as a individual element.

**Reply by saadeghi** [reply]
I will add it as an example. But I don't want to add a new class name for it. This whole `speed dial` concept was a Material Design idea and it's not really a "new element" it's just a button that shows more buttons. Some libraries like MUI provide it as a component because they are JS `<Component>` and they can insert multiple HTML tags into the Markup using one tag. daisyUI is just CSS class names.
